lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3516,6 +3516,20 @@ namespace {
             Impl.ImportedDecls.end())
           continue;
 
+        // If we encounter an IndirectFieldDecl, ensure that its parent is
+        // importable before attempting to import it because they are dependent
+        // when it comes to getter/setter generation.
+        if (const auto *Ind = dyn_cast<clang::IndirectFieldDecl>(nd)) {
```

Comment:
Nit: `camelCase` :P

Reply:
I actually misunderstood here, do you want Ind to be ind?

Reply:
Yes, exactly.